app/views/buyers/applications/_widget.html.erb

```diff
@@ -7,26 +7,26 @@
       <h2>Application</h2>
       <table class="list">
         <tr>
-          <th>Name</th>
+          <td>Name</td>
```

Comment:
Why change this?

Reply:
I guess because it's not cool to mix `th` and `td`. Although it looks that it's not against the rules: https://stackoverflow.com/questions/35608522/mixing-table-header-th-and-table-data-td-cells-in-a-single-table-row-tr

Reply:
Reverted